src/assets/DimensionItemIcons/PeriodIcon.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```

Comment:
In the src/assets directory there is a file called DynamicDimensionIcon.js. It makes me wonder how this folder should be organized - keep all dimension icons together in a single folder? Or keep all dimension icons directly under src/assets (e.g. PeriodDimensionIcon.js)

Reply:
The reason for the new folder was that there's a bunch of other icons (https://github.com/dhis2/design-system/tree/master/resources/icons/data-type) coming soon for the Data dimension modal (next in line), so I thought I'd prepare for those. They are very similar to the existing icons but we need both. However all of these will move til the ui-icons library in the near future regardless.